Sample/EventSourcing.Sample.Web/Controllers/AccountsController.cs

```diff
@@ -26,16 +27,18 @@ public AccountsController(ICommandBus commandBus, IQueryBus queryBus)
 
         // GET api/values
         [HttpGet]
-        public Task<IEnumerable<AccountSummary>> Get([FromQuery]GetAccounts query)
+        [Route("clientid={client_id}")]
+        public Task<IEnumerable<AccountSummary>> GetClientAccounts(Guid client_id)
```

Comment:
I think that it would be better to create ClientsContoller and move this method there so we could have endpoint like:
"clients/{clientId}/accounts ([reference](http://www.restapitutorial.com/lessons/restfulresourcenaming.html))